pkg/ambassador/tap/conduit/conduit.go

```diff
@@ -120,7 +120,7 @@ func (c *Conduit) Connect(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	logrus.Infof("Conduit connected: %v", c.Conduit)
+	logrus.Infof("Conduit connected: %v ; NSM Connection: %v", c.Conduit, c.connection)
```

Comment:
_c.connection_ is still nil here (either log _connection_ or move the printout)

Reply:
thank you, fixed